server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileBackuper.java

```diff
@@ -47,6 +47,8 @@
 
 public class QProfileBackuper implements ServerComponent {
 
+  public static final String MEDIA_TYPE_XML = "text/xml";
```

Comment:
Can be replaced by org.sonar.server.plugins.MimeTypes#XML. Note that this class could be moved in an appropriate package and that value is different ("application/xml"). It seems to have the same behavior than "text/xml" since HTTP 1.1.